org.siril.Siril.json

```diff
@@ -0,0 +1,288 @@
+{
+    "app-id": "org.siril.Siril",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "command": "siril",
+    "rename-icon": "org.siril.Siril",
+    "rename-mime-file": "siril.xml",
+    "rename-mime-icons": [
+        "text-x-seq"
+    ],
+    "provides": {
+        "id": "org.free_astro.siril"
+    },
+    "replaces": {
+        "id": "org.free_astro.siril"
+    },
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        /* Needed for gvfs to work */
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfs",
+        "--filesystem=xdg-run/gvfsd",
+        "--share=network"
+    ],
+    "modules": [
+        {
+            "name": "cfitsio",
+            "config-opts": [
+                "--enable-reentrant"
+            ],
+            "make-args": ["shared"],
+            "cleanup": [
+                "/include",
+                "/lib/*.a",
+                "/lib/*.la",
+                "/lib/pkgconfig"
+            ],
```

Comment:
Can you combine the common cleanups into a toplevel one ?

`/lib/*.a` -> `"*.a" and `/lib/*.la` -> `"*.la"`

Reply:
fixed in 57d8a6a